SETUP.md

```diff
@@ -61,6 +61,21 @@ Assuming the repo is cloned as `Recommenders` in the local system, to install **
 
 You can specify the environment name as well with the flag `-n`.
 
+<details>
+<summary><strong><em>Java version for PySpark</em></strong></summary>
+
+**Note.** By default we use pyspark v2.4.3. It doesn't work on Java versions >8.
```

Comment:
@yegorkryukov thanks for the contribution. In linux we are using 2.4.5, [see this](https://github.com/microsoft/recommenders/blob/master/tools/generate_conda_file.py#L137). Does 2.4.5 work with Java 8 on Mac?

Reply:
@miguelgfierro I'm not sure. My `reco_pyspark` environment uses Spark 2.4.3 and it works with Java 8.

Reply:
I just installed 2.4.5 and now pyspark won't work.

Reply:
wow, not very stable :-)

would 2.4.5 work if you install java 9?

Reply:
Nope. Didn't work with Java 9 either.

Reply:
@yegorkryukov I just tested on my mac (Catalina 10.15.7) with Spark 2.4.5 + Java 8 and I could run als_movielens notebook w/o any issues.
Few things we can improve the installation steps in this PR are:

* JAVA8 installation on MacOS is relevant to readers only when they install pyspark. We can move the section under **PySpark environment** installation section. See **Set PySpark environment variables on Linux or MacOS** section e.g.. We can move JAVA8 installation section right before **Set PySpark environment variables on Linux or MacOS**. 
* At the end of JAVA8 installation instruction, let readers verify installation via `java -version`. Also, one may still use bash. We can simply add a comment: `# in bash, run: . ~/.asdf/plugins/java/set-java-home.bash` at the end of the instruction script.
* @miguelgfierro I think we should remove SPARK_HOME (maybe we backup the path at env-activation and recover when deactivate env. I recall we had that codes before). If we set SPARK_HOME to spark installation path, pyspark uses that installed version instead of the version we installed (2.4.5).